.github/workflows/seed-tables.yaml

```diff
@@ -0,0 +1,72 @@
+name: 🌱 Seed Tables
+run-name: 🌱 ${{ github.actor }} is seeding ${{ inputs.environment || github.ref_name }}
+on:
+  workflow_dispatch:
+    inputs: 
+      environment: 
+        description: The environment to seed (if different from selected branch)
+        type: string
+        required: false
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+  prepare:
+    name: 🧹 Prepare Seeds
+    runs-on: ubuntu-latest
+    outputs: 
+      github-branch: ${{ steps.environment.outputs.github-branch }}
+      environment-name:  ${{ steps.environment.outputs.environment-name }}
+      environment-type:  ${{ steps.environment.outputs.environment-type }}
+      seed_tables:  ${{ steps.get_seed_tables.outputs.value }}
+    steps:
+
+      - name: ↕️ Get Environment Details
+        id: environment
+        uses: dvsa/cvs-github-actions/environment@feature/CB2-10422
+        with:
+          environment: ${{ inputs.environment || github.ref_name }}
+
+      - name: 📥 Get Codebase
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.environment.outputs.github-branch }}
+          
+      - name: 📥 Get Seed Tables
+        id: get_seed_tables
+        run: |
+          echo "value=$(cat seed-tables.json | tr -d ' \n')" >> $GITHUB_OUTPUT
+
+  seed_table:
+    name: 🌱 Seed Table
+    needs: prepare
+    runs-on: [self-hosted, X64]
```

Comment:
Why use both? self-hosted covers all the X64 runners as well as the m2 macs.

Reply:
Belt-and-braces - the runs-on needs to match all of the values (so will _only_ run on a self-hosted node with the X64 flag)

It's just to remove any potential ambiguity in case the tag changes in future, or for some inexplicable reason Github decide to create a runner that matches our self-hosted tag.

If, for instance, we spin up a ubuntu-latest container/VM on the Macs, adding the `self-hosted` ensures it's not going to cause a conflict.